src/lxml/html/tests/test_diff.txt

```diff
@@ -54,8 +54,8 @@ As a special case, changing the href of a link is displayed, and
 images are treated like words:
 
     >>> pdiff('<a href="http://yahoo.com">search</a>', '<a href="http://google.com">search</a>')
-    <a href="http://google.com">search <ins> Link: http://google.com</ins>
-    <del> Link: http://yahoo.com</del> </a>
+    <a href="http://google.com">search <ins> Link: http://google.com
+    </ins> <del> Link: http://yahoo.com</del> </a>
```

Comment:
Was this a necessary change? It doesn't feel right.

Reply:
tbh, didn't feel right to me either but this test breaks without it, and I didn't have much luck trying to figure out why. Initial instinct was that it has something to do with the `re.sub` in the `pwrapped` definition at the top of the test file (possibly the `+` in the re over matches ?). Apologies, I didn't try digging deeper but I'll give it a shot now.

Reply:
So the underlying cause for this appears to be an extra trailing whitespace that appears to get added to the `href_token` which pushes the line length of the string to over 70 (the line length limit of textwrap, used by pdiff). Here are the diffs with the current lxml code and one taken with this patchset:

current lxml diff:

`<a href="http://google.com">search <ins> Link: http://google.com</ins> <del> Link: http://yahoo.com</del> </a>`

diff with this patchset:
`<a href="http://google.com">search <ins> Link: http://google.com </ins> <del> Link: http://yahoo.com</del> </a>`

What I'm not entirely sure about is why the trailing whitespace which ideally *should* also be present in the current lxml diff (according to call to `href_token` in `fixup_chunks`), isn't.

In any case, would you consider this as a regression in behavior worthy of a fix or does this fix of the test case suffice ?

Reply:
What strikes me is that there is a space before the closing `</ins>` but not before the closing `</del>`. Given that there is a space after both of the opening `<ins>` and `<del>`, it's debatable whether the new behaviour isn't really better than the old. But I would expect both tag types to be consistent, at least.

Reply:
I agree. I'll investigate and attempt to make them consistent.

Reply:
I've addressed this issue by removing the inclusion of the trailing whitespace during the creation of `href_token` instance which adds the text `Link: ...`. The reasoning for this is that the only time this piece of string would be displayed would be when there's a change, which necessarily means it would be handled in the `merge_insert` function.

By making this change, I no longer needed to update the existing test case.